lib/vfscore/Config.uk

```diff
@@ -29,7 +29,7 @@ config LIBVFSCORE_NONLARGEFILE
 		the 64-bit version of the system calls are registered.
 
 config LIBVFSCORE_AUTOMOUNT_ROOTFS
-bool "Automatically mount a root filesysytem (/)"
+bool "Automatically mount a root filesystem (/)"
```

Comment:
Thanks for this fix :+1:. A few minor comments:
- Can you update your commit title to use `lib/vfscore: Fix spelling mistake in Config.uk`, too? I think this is better readable with `git log`.
- Can you address the other issues that checkpatch complains about in your commit message (mostly line length)?
- We use the `Signed-off-by` tag as an act of signing your contribution that it complies with the [Certificate of Origin](https://unikraft.org/docs/contributing/unikraft#developers-certificate-of-origin). Would you be able to use a proper mail address of yours which is not a "noreply" address?

Reply:
Hello @skuenzer 
I have done the following changes, kindly review them once again. Thank you!